docs/changehistory/NextVersion.md

```diff
@@ -3,3 +3,19 @@ publish: false
 ---
 # NextVersion
 
+(??)## Hiliting models and subcategories
```

Comment:
What's up with all the `??`s?

Reply:
If you're wondering where to put it, can you put these notes in the `3.2.0.md` file? That will just require a new docs deployment to get added in.

Reply:
Thanks for catching that. I am using a different diff tool to resolve conflicts on linux, it inserts those.

Reply:
Ok I wondered about NextVersion - will move to 3.2.0.md.

Reply:
Moved.